.github/workflows/sync-upstream.yml

```diff
@@ -0,0 +1,35 @@
+name: Update
+
+on:
+  # Runs at 10:00 UTC every day
+  schedule:
+    - cron:  '0 10 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  sync:
+    name: Sync version with upstream
+    environment: "Candidate Branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Sync version with upstream
+        uses: snapcrafters/ci/sync-version@main
+        with:
+          token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
+          update-script: |
+            stable_version=$(curl -sL https://www.sublimemerge.com/download | grep -Po '<p class="latest"><i>Version:</i> Build \K[0-9]+')
+            yq -i ".version = \"$stable_version\"" snap/snapcraft.yaml
+
+            # Support for dev will be added later
+            # dev_version=$(curl -sL https://www.sublimemerge.com/dev | grep -Po '<p class="latest"><i>Version:</i> Build \K[0-9]+')
+            # if [[ "$stable_version" > "$dev_version" ]]; then
+            #     yq -i ".version = \"$stable_version\"" snap/snapcraft.yaml
+            # else
+            #     yq -i ".version = \"$dev_version-dev\"" snap/snapcraft.yaml
+            # fi
+            
```

Comment:
Would be good to tidy up the whitespace here and just end the file with a single newline

Reply:
sure